pinot-transport/src/main/java/com/linkedin/pinot/routing/builder/KafkaLowLevelConsumerRoutingTableBuilder.java

```diff
@@ -55,6 +55,7 @@ public void init(Configuration configuration) {
       final String targetServerCountPerQuery = configuration.getString("realtimeTargetServerCountPerQuery");
       try {
         TARGET_SERVER_COUNT_PER_QUERY = Integer.parseInt(targetServerCountPerQuery);
+        LOGGER.info("Using target server count of {}", TARGET_SERVER_COUNT_PER_QUERY);
       } catch (Exception e) {
         LOGGER.warn(
             "Could not get the target server count per query from configuration value {}, keeping default value {}",
```

Comment:
canu modify this log as well to include the info that this is for realtime routing tables?